pages/help/glossary.mdx

```diff
@@ -0,0 +1,91 @@
+## General 
+
+| Acronym/Abbreviation         | Definition                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| ACL                          | Access Control List                                                                      |
+| API                          | Application Programming Interface                                                        |
+| ARPA                         |  Accountable, Responsible, Participant, Advisor                                          |
+| BAM                          | Bidirectional Associative Memory                                                         |
+| BCR                          | Biospecimen Core Resource                                                                |
+| CI/CD                        |                                                                                          |
+| CNV                          |  Copy Number Variation                                                                   |
+| DCA                          | Data Curator App                                                                         |
+| DCC                          | Data Coordinating Center                                                                 |
+| eRA                          | Electronic Research Administration                                                       |
+| FISH                         |                                                                                          |
+| GCA                          | Genomic Data Commons                                                                     |
```

Comment:
GDC -- this and NIH, NCI, TCGA, and even HTAN can be moved to something like "Recommended/Related Research Orgs"?